presto-hive-hadoop2/bin/common.sh

```diff
@@ -45,8 +45,11 @@ function exec_in_hadoop_master_container() {
 function stop_unnecessary_hadoop_services() {
   HADOOP_MASTER_CONTAINER=$(hadoop_master_container)
   docker exec ${HADOOP_MASTER_CONTAINER} supervisorctl status
-  docker exec ${HADOOP_MASTER_CONTAINER} supervisorctl stop yarn-resourcemanager
-  docker exec ${HADOOP_MASTER_CONTAINER} supervisorctl stop yarn-nodemanager
+
+  if [[ ${TESTS_HIVE_VERSION_MAJOR} -lt 3 ]]; then
```

Comment:
Why was this necessary? I don't see it in Trino.

Reply:
These two services are started only for Hive < 3. For the new Hive 3.1.2 image, these services won't be started, so trying to stop them throws an error. My guess is Trino is not using Apache Hive 3.1.2 image for tests. Otherwise, they would have faced this problem too.